security/pfSense-pkg-openvpn-client-export/files/usr/local/pkg/openvpn-client-export.inc

```diff
@@ -991,13 +991,10 @@ function openvpn_client_export_find_port_forwards($targetip, $targetport, $targe
 
 	foreach ($config['nat']['rule'] as $natent) {
 		$dest = array();
-		if (!isset($natent['disabled']) && ($natent['target'] == $targetip) && ($natent['local-port'] == $targetport) && ($natent['protocol'] == $targetproto)) {
-			$dest['proto'] = $natent['protocol'];
-
-			// Could be multiple ports... But we can only use one.
-			$dports = is_port($natent['destination']['port']) ? array($natent['destination']['port']) : filter_expand_alias_array($natent['destination']['port']);
-			$dest['port'] = $dports[0];
-
+                $proto_short = strtolower(substr($targetproto, 0, 3));
```

Comment:
Since this variable is only used once, skip setting it this way and use the contents in the next line directly instead.